### PR TITLE
Completed implementation and preliminary unit tests for the MP evalua…

### PR DESCRIPTION
**Pull request by sanleyCVP**
…tion report

## Pull Request

```
2275 MP Evaluation Report layout completed

```

### CODE REVIEW

File: src/components/MonitoringPlanEvaluationReport/MonitoringPlanEvaluationReport.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useRef } from "react";
+import { Button } from "@trussworks/react-uswds";
+import { DataTableRender } from "../DataTableRender/DataTableRender";
+
+
+export const MonitoringPlanEvaluationReport = ({
+}) => {
```
Comment (tdavydets) [inline src/components/MonitoringPlanEvaluationReport/MonitoringPlanEvaluationReport.js]: the {} should be taken out of the arguments if not being used (it'll flag it as a bug)

File: src/components/HeaderInfo/HeaderInfo.js [hunk 2]
```diff
@@ -450,8 +347,8 @@ export const HeaderInfo = ({
                         onClick={() => checkoutStateHandler(true)}
                         id="checkOutBTN"
                         epa-testid="checkOutBTN"
-                        //508
-                        // ref={checkout ? activeFocusRef : null}
+                      //508
```
Comment (tdavydets) [inline src/components/HeaderInfo/HeaderInfo.js:350]: please remove comments if not using

File: src/components/HeaderInfo/HeaderInfo.test.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+import React from "react";
+import {
+  render,
+  screen,
+  within,
+  fireEvent,
+  waitForElement,
+} from "@testing-library/react";
+import HeaderInfo from "./HeaderInfo";
 // // import { getActiveConfigurations } from "../../utils/selectors/monitoringConfigurations";
```
Comment (tdavydets) [inline src/components/HeaderInfo/HeaderInfo.test.js:10]: please remove commented out content if not using

File: src/components/HeaderInfo/HeaderInfo.test.js [hunk 4]
```diff
@@ -1153,3 +1153,48 @@ test("test file", () => {
...
+      sectionSelect={[4, "Methods"]}
+      inactive={[true, false]}
+      locations={[{ id: "6", name: "1", type: "Unit", active: true }]}
+    />
+  )
+
+  expect(container.instance().setAuditInformation()).toContain("Currently checked out by:");
+});
+
+// test("test audit records for workspace checked in", () => {
```
Comment (tdavydets) [inline src/components/HeaderInfo/HeaderInfo.test.js:1176]: please remove commented out test if not using

File: src/components/MonitoringPlanEvaluationReport/MonitoringPlanEvaluationReport.js [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+        col2: "Severity 1",
+        col3: "category 1",
+        col4: <Button
+            type="button"
+            unstyled={true}
+            className="text-primary text-underline"
+            href="#"
+            role="link"
+            rel="Check Code 1"
+        // title={`Go to ${ele.name} page`}
```
Comment (tdavydets) [inline src/components/MonitoringPlanEvaluationReport/MonitoringPlanEvaluationReport.js]: please remove commented out content if not using

**Review by tdavydets — CHANGES_REQUESTED** [review]

File: src/components/MonitoringPlanEvaluationReport/MonitoringPlanEvaluationReport.js (hunk 1 above)
Comment (sanleyCVP) [reply src/components/MonitoringPlanEvaluationReport/MonitoringPlanEvaluationReport.js]: Fixed, with latest push

File: src/components/MonitoringPlanEvaluationReport/MonitoringPlanEvaluationReport.js (hunk 5 above)
Comment (sanleyCVP) [reply src/components/MonitoringPlanEvaluationReport/MonitoringPlanEvaluationReport.js]: Removed

File: src/components/HeaderInfo/HeaderInfo.test.js (hunk 4 above)
Comment (sanleyCVP) [reply src/components/HeaderInfo/HeaderInfo.test.js:1176]: Used as reference for pending updates to unit tests

File: src/components/HeaderInfo/HeaderInfo.test.js (hunk 3 above)
Comment (sanleyCVP) [reply src/components/HeaderInfo/HeaderInfo.test.js:10]: Used as reference for pending updates to unit tests

**Review by tdavydets — APPROVED** [review]

**Pull request merged**